generic/secrets/security/detected-azure-service-bus-key.yaml

```diff
@@ -0,0 +1,9 @@
+rules:
+- id: detected-azure-service-bus-key
+  pattern-regex: |-
+    [A-Za-z0-9\/+-]{43}
```

Comment:
If I read this regex correctly, this will detect a lot of high entropy strings that may not be an Azure service bus key. Is there a way to modify the regex or use a `pattern-not-regex` to better detect when this is an Azure service bus key?